IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -113,38 +114,103 @@ - (void)acquireTokenWithRequest:(MSIDSilentTokenRequest *)request
 
     CONDITIONAL_START_EVENT(CONDITIONAL_SHARED_INSTANCE, self.requestParameters.telemetryRequestId, MSID_TELEMETRY_EVENT_API_EVENT);
     self.currentRequest = request;
+    __weak typeof (self) weakSelf = self;
```

Comment:
Do we need a weak reference here? "request" was created locally and seems doesn't create the ref cycle.

Reply:
Updated.

Originally I thought the ```self.currentRequest = request;``` and there created a strong ref. But after double-checking the instruments, no memories leaks even without weak/strong dance.